complex/ft_test.c

```diff
@@ -262,10 +262,55 @@ static int ft_sync_test(int value)
 	return ft_sock_sync(value);
 }
 
+static int ft_pingpong_rma(void)
+{
+	int ret, i;
+	size_t msg_size = ft_tx_ctrl.msg_size;
+
+	if (listen_sock < 0) {
+		for (i = 0; i < ft_ctrl.xfer_iter; i++) {
+			ft_tx_ctrl.msg_size = msg_size;
+			ret = ft_send_rma();
+			if (ret)
+				return ret;
+
+			ft_tx_ctrl.msg_size = ft_ctrl.size_array[0];
+			ret = ft_send_msg();
```

Comment:
Note that this can result in the send being transferred before the read completes, which can get the tests out of sync and avoid doing a true ping-pong.  For read testing, we need to get the read completion before sending the sync message.